programs/degen-pools/src/pool_options.rs

```diff
@@ -8,6 +8,12 @@ pub struct PoolOption {
     pub title: String,
     pub value: u64,
 }
+
+#[event]
+pub struct OptionCreated {
+    pub pool_account: Pubkey,
```

Comment:
Let's include the option title in the event so the backend can store it also

Reply:
Done